python/lsst/daf/persistence/test/testMapper.py

```diff
@@ -21,11 +21,37 @@
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
 
+import os
+
+from .. import ButlerLocation
 from .. import Mapper
+from .. import Storage
+from . import TestObject
 
 
 class EmptyTestMapper(Mapper):
     """Class that can be used as a stub for a mapper."""
 
-    def __init__(self, root=None, parentRegistry=None, repositoryCfg=None):
+    def __init__(self, root=None, parentRegistry=None, repositoryCfg=None, **kwargs):
+        self.kwargs = kwargs
         pass
+
+
+class MapperForTestWriting(Mapper):
+
+    def __init__(self, root, **kwargs):
+        self.root = root
+        self.storage = Storage.makeFromURI(self.root)
+
+    def map_foo(self, dataId, write):
+        python = TestObject
+        persistable = None
+        storage = 'PickleStorage'
+        fileName = 'filename'
+        for key, value in dataId.items():
```

Comment:
Maybe `sorted(dataId.items())` to make sure that file name does not depend on the order of the items returned from `items()` (I think that order is not guaranteed to be identical even if content of dict is identical though maybe I'm confusing it with something else)?